content/docker-hub/access-tokens.md

```diff
@@ -65,10 +65,16 @@ a password.
 You can rename, activate, deactivate, or delete a token as needed.
 
 1. Access your tokens under **Account Settings > Security**.
-   This page shows an overview of all your tokens. You can also view the number
+   This page shows an overview of all your tokens, and lists if the token was generated manually or if it was [auto-generated](#auto-generated-tokens). You can also view the number
    of tokens that are activated and deactivated in the toolbar.
 
-2. Choose a token and then select **Delete** or **Edit**, or use the menu on the far right of a token row to bring up the edit screen. 
+2. Select a token and then select **Delete** or **Edit**, or use the actions menu on the far right of a token row to bring up the edit screen.
    You can also select multiple tokens to delete at once.
 
-3. After modifying the token, select **Save**.
\ No newline at end of file
+3. After modifying the token, select **Save**.
+
+## Auto-generated tokens
+
+When you sign in to Docker Desktop, this generates an access token on your behalf. When you complete registry actions using the Docker CLI, the CLI uses this token for authentication. The token scope has Read, Write, and Delete access. If your Docker Desktop session expires, the token is automatically removed. You can have up to 5 auto-generated tokens associated with your account. These are deleted and created automatically based on usage and creation dates.
```

Comment:
@technicallyjosh I included `If your Docker Desktop session expires, the token is automatically removed.` because i had it in my notes from our meeting, but is this accurate? I tested out logging out and closing DD still see the same token active. Is there more context here for what session expiration means? I can also remove this line if it's more of a behind-the-scenes action not necessary to include.

Reply:
Yeah this is true. Currently this is only when it automatically expires. Not when you "log out". I'll talk to the desktop team about that one. I'm not sure I'd remove it. It could be a "info" piece.

Reply:
Additional context here is that we remove it from local keychain/secure storage. We do not delete it from the account.

Reply:
@technicallyjosh I made a small update based on that info from above. I also moved down the info about the max tokens so that the account-related token management is in one paragraph. Let me know if this works!

Reply:
Looks like the deploy failed along with some checks. Can't view till that's working 😬.

Reply:
Yeah the Netlify checks were having an issue. Re-pushed so you should see the preview now!